.kiro/specs/auth-controller-refactoring/tasks.md

```
@@ -1321,28 +1496,51 @@ This phase enables the refactored endpoints by setting feature flags in `appsett
 4. ✅ **Milestone 4**: Controller modified (End of Phase 4)
 5. ✅ **Milestone 5**: Ready for deployment (End of Phase 5)
 6. ✅ **Milestone 6**: All endpoints at 100% rollout (End of Phase 6)
-7. ✅ **Milestone 7**: Legacy code removed (End of Phase 7)
-8. ✅ **Milestone 8**: JSON API complete (End of Phase 8)
-9. ✅ **Milestone 9**: Rendering abstracted (End of Phase 9)
-10. ✅ **Milestone 10**: Next.js frontend live (End of Phase 10)
-11. ✅ **Milestone 11**: Frontend-agnostic architecture complete (End of Phase 11)
+7. ✅ **Milestone 7**: Legacy code cleaned up (End of Phase 7) - **STOPPING POINT FOR MOST TEAMS**
+
+**Future Vision (Optional - Choose Your Path)**:
+8. ⭕ **Milestone 8**: JSON API complete (End of Phase 8) - *Option 2 or 3*
```

markdownlint-cli2 (0.21.0)

[warning] Emphasis style
Expected: underscore; Actual: asterisk

(MD049, emphasis-style)